db/data/20231018062055_re_enable_sms_india.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+class ReEnableSmsIndia < ActiveRecord::Migration[6.1]
```

Comment:
```suggestion
class ReduceSmsBatchSizeIndia < ActiveRecord::Migration[6.1]
```

Reply:
I'm rewording it to SetupSMSExperimentsInIndia, since this data migration now has new filters to update.